examples/data-objects/todo/src/Todo/Todo.tsx

```diff
@@ -80,13 +83,21 @@ export class Todo extends DataObject implements IFluidHTMLView {
         // Create a new todo item
         const component = await TodoItem.getFactory().createChildInstance(this.context, props);
 
-        // Store the handle to the component in the sequence
-        this.todoItems.insert(this.todoItems.getLength(), [component.handle]);
+        // Generate a key that we can sort on later, and store the handle.
+        this.todoItemsMap.set(`${Date.now()}-${uuid()}`, component.handle);
```

Comment:
we should put the timestamp in the item, so {handle, timestamp}. it doesn't matter much is this case, but if you wanted to reorder this would be the way, as changes to the sort key, timestamp in this case, would coalesce.  Even better than a single object in a single map key would be a subdirectory per entry

Reply:
i just don't want the sort by map key in our example, as it prevents move

Reply:
similarly, we could revert this, and open an issue to correctly fix it

Reply:
Updated in latest to store the index in the map value rather than sorting on the key.

I am eager to eliminate usage of `SharedObjectSequence` to prevent copycatting so don't want to revert, but acknowledge that there will be better solutions.  I think we should let this scenario help drive design ideas in #8526.